### PR TITLE
Updating contingency lists sends an 'element.update' notification

### PR DESCRIPTION
**Pull request by archendel**
Signed-off-by: LE SAULNIER Kevin <kevin.lesaulnier@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/actions/server/NotificationService.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package org.gridsuite.actions.server;
```
[the remark this answers is not in the document]
Comment (archendel) [reply src/main/java/org/gridsuite/actions/server/NotificationService.java:8]: fixed

File: src/main/java/org/gridsuite/actions/server/NotificationService.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Service;
+
+@Service
```
[the remark this answers is not in the document]
Comment (archendel) [reply src/main/java/org/gridsuite/actions/server/NotificationService.java:25]: fixed

File: src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java [hunk 3]
```diff
@@ -865,7 +920,8 @@ public void copyFormToScriptTest() throws Exception {
         UUID firstUUID = addNewFormContingencyList(form);
 
         // new script from form
-        String res = mvc.perform(post("/" + VERSION + "/form-contingency-lists/" + firstUUID + "/new-script"))
+        String res = mvc.perform(post("/" + VERSION + "/form-contingency-lists/" + firstUUID + "/new-script")
+                .header(USER_ID_HEADER, "userId"))
```
Comment (Meklo) [inline src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java:924]: Why is userId needed here ? newScriptFromFormContingencyList doesn't request it in the controller

Comment (archendel) [reply src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java:924]: removed

**Review by Meklo — APPROVED** [review]

**Review by niconoir — APPROVED** [review]

**Pull request merged**